vnext/Microsoft.ReactNative/Views/TextInputViewManager.cpp

```diff
@@ -308,6 +309,10 @@ void TextInputShadowNode::registerEvents() {
       });
 
   m_controlLoadedRevoker = control.Loaded(winrt::auto_revoke, [=](auto &&, auto &&) {
+    if (m_autoFocus) {
```

Comment:
Not sure, but I think we might prefer to use XAML's focus manager, rather than control.Focus()
@asklar Do you remember any discussion about this or whether one way is preferred?

Reply:
I assumed FocusManager was used in the UIManager so you could call it on an arbitrary DependencyObject. In this case, we know what we have is a Control, so I figured we might as well use the Focus method directly.

Reply:
That makes total sense, I just vaguely remember some focus mechanism being deprecated, so wanted to make sure,